CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ option(USE_VCPKG "Use VCPKG to download and manage dependencies" OFF)
 
 if (USE_VCPKG)
     # Detect correct triplet based on CMake env
-    if (WIN32)
+    if (CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
```

Comment:
I think there is an `EMSCRIPTEN` variable in Cmake no?

Reply:
The variable `EMSCRIPTEN` was not available here.  Maybe I'm missing something when configuring cmake?